MojioSDK/Client/Vehicles/VehiclesClient.swift

```diff
@@ -324,11 +324,13 @@ open class VehiclesClient: RestClient {
             case .timeline:
                 fallthrough
             case .activities:  // conformance to be implemented
+                let decoder = JSONDecoder()
+                decoder.dateDecodingStrategy = .formatted(VehiclesClient.dateFormatterIso8601)
```

Comment:
The Activity decoder already handles ISO dates (See: dateFromISO) including inconsistent formats from the platform so this isn't needed.